include/zephyr/bluetooth/conn.h

```diff
@@ -323,6 +323,42 @@ enum bt_conn_state {
 	BT_CONN_STATE_DISCONNECTING,
 };
 
+/** Security level. */
+typedef enum __packed {
+	/** Level 0: Only for BR/EDR special cases, like SDP */
+	BT_SECURITY_L0,
+	/** Level 1: No encryption and no authentication. */
+	BT_SECURITY_L1,
+	/** Level 2: Encryption and no authentication (no MITM). */
+	BT_SECURITY_L2,
+	/** Level 3: Encryption and authentication (MITM). */
+	BT_SECURITY_L3,
+	/** Level 4: Authenticated Secure Connections and 128-bit key. */
+	BT_SECURITY_L4,
+	/** Bit to force new pairing procedure, bit-wise OR with requested
+	 *  security level.
+	 */
+	BT_SECURITY_FORCE_PAIR = BIT(7),
+} bt_security_t;
+
+/** Security Info Flags. */
+enum __packed bt_security_flags {
+	/** Paired with Secure Connections. */
+	BT_CONN_FLAG_SC = BIT(0),
```

Comment:
The enum name and the enum value prefix should be the same, e.g. either rename enum to `bt_conn_flag` or the enums to `BT_SECURITY_FLAGS`